Changes to examine

comma opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Clothing now hides wounds. If it's not thick (like spacesuits etc), blood would still show through it for bleeding wounds.
Fixed dislocation check waiting for wrong value.
Merged two organ loops in one. Cleaned up it slightly.

Fixes https://github.com/Baystation12/Baystation12/issues/14139

comma committed:
Changes to examine

Clothing now hides wounds. If it's not thick (like spacesuits etc), blood would still show through it for bleeding wounds.
Fixed dislocation check waiting for wrong value.
Merged two organ loops in one. Cleaned up it slightly.

Ithalan commented:
It seems counter intuitive that you can't figure out if parts of your body are physically wounded unless you can see them directly. The pain and general effect on sense of touch in the affected area would give not only a decent idea of where they are, but also whether it is a burn, bruise or cut.

I would suggest making it so that clothes hiding wounds only applies when examining others.

comma commented:
You can still click on yourself with help intent to get readout, but yeah, makes sense.

comma committed:
Adds personal xray vision

I guess you can notice missing limbs even if you cover them up with a spacesuit.

mustafakalash commented:
Surely if you were burned, cut, or shot, that would have had to happen through your clothes, and would be visible. Unless you change clothes I guess, but that's hardly practical to implement.

PsiOmegaDelta commented:
Cannot have everything in this world.
:+1:

mustafakalash commented:
I don't see the purpose of the change. It isn't more realistic (not that that's a good reason), and it's a tiny nerf that we don't need.

comma commented:
That's how it used to work, got lost somewhere along the way. The wounds that need immediate attention are still visible, the rest still visible on health analyzers.

SparklySheep commented:
Plus you can also grab someone and then click them with health intent to inspect for wounds.

ghost commented:
I love having an excuse to use the grab-examine, and personally think it should be used more often- this minor nerf means it would be, so I'm supportive of this.
